README.md

```diff
@@ -1,2 +1,29 @@
-# agatedb
-A persistent key-value storage in rust.
+# AgateDB
+
+AgateDB is an embeddable, persistent and fast key-value (KV) database written
+in pure Rust. It is designed as an experimental engine for The [TiKV][1]
+project that will bring aggressive optimizations for TiKV specifically.
+
+## Project Status
+
+AgateDB is still under early heavily development, you can check the development
+progress at the [Project][2].
+
+The whole plan is to port [badger][3] in Rust first and then port the
+optimizations that have been made in [unistore][4].
+
+[1]: https://github.com/tikv/tikv
+[2]: https://github.com/tikv/tikv/projects/1
+[3]: https://github.com/dgraph-io/badger
```

Comment:
consider giving a specific commit, so we can review based on a consistent base.

Reply:
What about using current latest master? https://github.com/dgraph-io/badger/tree/45bca18f24ef5cc04701a1e17448ddfce9372da0